src/odemis/gui/util/img.py

```diff
@@ -1205,9 +1207,19 @@ def spectrum_to_export_data(proj, raw, vp=None):
         ctx = cairo.Context(surface)
 
         if vp is not None:
+
             # Limit to the displayed ranges
             range_x = vp.hrange.value
             range_y = vp.vrange.value
+            # extend the range_y by 5% from both sides (range extended by 10% in total)
+            ext = (range_y[1] - range_y[0]) * 0.05
+            # Check if the range includes 0, after the extension, and compare 0 with the initial minimum and maximum
+            # values to replace them if needed.
```

Comment:
I think this could be a single comment, as it's all about the same thing. Something like:
"Check if the range is near 0, and if so, extend it slightly to include it.
We define "near" as 5% of the range."

BTW, I think that 10% would be very acceptable (eg, if values between 1 and 9 => extending to 0 and 9 would be acceptable).